usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -193,7 +193,7 @@ def create_recipient_object(db_row_dict: dict) -> OrderedDict:
                         ("country_name", db_row_dict["_rl_country_name"]),
                         ("state_code", db_row_dict["_rl_state_code"]),
                         ("state_name", db_row_dict["_rl_state_name"]),
-                        ("city_name", db_row_dict["_rl_city_name"]),
+                        ("city_name", db_row_dict["_rl_city_name"] or db_row_dict["_rl_foreign_city"]),
```

Comment:
We are still returning the foreign city name in `city_name` correct?

Reply:
yes